README.md

```diff
@@ -95,10 +95,15 @@ deploying and running an example chaincode.
 ## Requirements
 
 * CMake 3.5.1 or higher
-* Go 1.11 or higher
-* Install Fabric v1.4 https://github.com/hyperledger/fabric 
-* Install the Linux SGX SDK v2.4 https://github.com/intel/linux-sgx 
-* You also need to install Intel SGX SSL https://github.com/intel/intel-sgx-ssl
+* Go 1.11.x or higher
+* Hyperledger Fabric v1.4 https://github.com/hyperledger/fabric 
+
+    If you are new to Fabric, we recommend the Fabric documentation as your starting point. You should start with
+    [installing](https://hyperledger-fabric.readthedocs.io/en/release-1.4/prereqs.html) Fabric dependencies and setting
+    up your [development environment](https://hyperledger-fabric.readthedocs.io/en/release-1.4/dev-setup/build.html).
+    
+* SGX SDK v2.4 for Linux https://github.com/intel/linux-sgx 
```

Comment:
btw: you hard-coded now 2.4 in a patch for docker-sgx as that didn't seem to work anymore with 2.5. However, docker-sgx now fixed the docker file such that master with your patch does _not_ work anymore.  Maybe for now easiest would be to update docu to say one should check out commit is08c2457 for docker-sgx (bruno tested that config).  For more long-term i propose we eventually dump docker-sgx and go with our own.  We could already use the one from PDO but probably better would be to do the switch once i have a docker image for DCAP (which still supports epid) which i plan to do for PDO and try to do it also in a way that it would be easy shareable/copyable for TCC ...

Reply:
Thank you. Let's create a separate issue for this problem. I also agree that we should maintain our own dockerfile and thereby remove the dependency of another project. 
Updating to SGX 2.5 is yet another TODO.
Can you approve and merge?